presentation-api/receiver/index.html

```diff
@@ -17,15 +17,81 @@
         left: 50%;
         transform: translate(-50%, -50%);
         text-align: center;
-        font-size: 75px;
+        font-size: 150px;
       }
     </style>
+    <script type="text/javascript">
```

Comment:
We should add this as a separate script we could link from the sample so that developers understand how works communication.